zipline/__main__.py

```diff
@@ -251,6 +251,12 @@ def run(ctx,
 def zipline_magic(line, cell=None):
     """The zipline IPython cell magic.
     """
+    load_extensions(
+        default=True,
+        extensions=[],
+        strict=True,
+        environ=os.environ,
+    )
     try:
         return run.main(
```

Comment:
It does not appear to get called, I think this would work if I did `main.main`, but I don't want users to need to pass the subcommands.